src/textfully.ts

```diff
@@ -1,4 +1,4 @@
-import {
+import type {
```

Comment:
This ensures that a bundler would drop this file entirely since it's only used to import types